emails/2024-04-26.md

```diff
@@ -0,0 +1,31 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+I've been real sick this week but that's not going to stop me from cranking out the weekly email. I've never heard of anyone catching a cold from an email.
+
+### Dictionary Compression
+
+One of the objections we hear to using Dolt is, "I can't possibly store all the history of my database in my database". [Neil](https://www.dolthub.com/team#neil) is on a mission to address this concern. 
+
+First up is better compression for Dolt storage. Neil goes into detail [in his latest blog](https://www.dolthub.com/blog/2024-04-22-dolt-storage-dictionaries/) on how dictionary compression works and how it can compress Dolt databases by about 50% from current footprints. That's an exciting win especially when you consider the clone use case. Get a database from DoltHub twice as fast. Yes, please.
+
+[![Compression Ratios](../images/compression-ratios.png)](https://www.dolthub.com/blog/2024-04-22-dolt-storage-dictionaries/)
+
+Next, Neil will follow up his [shallow clone work](https://www.dolthub.com/blog/2024-02-21-shallow-clone/) with a project we call history offloading where you can store the last N commits locally and have the rest of your history stored on [a remote](https://www.dolthub.com/blog/2024-04-25-why-remotes/), like DoltHub or DoltLab. 
+
+We think with these two changes, most users will feel comfortable storing the history of their database in their database using Dolt.
```

Comment:
end of this sentence is hard to parse. How about:

With these two changes Dolt users can keep as much or as little of their history as they choose. How much history will you save? 1 month, 1 year, eternity? Up to you!

Reply:
I like it